rclcpp/include/rclcpp/parameter_client.hpp

```diff
@@ -136,9 +137,9 @@ class AsyncParametersClient
   std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>
   set_parameters(
     std::vector<rclcpp::parameter::ParameterVariant> parameters,
-    std::function<
-      void(std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>)
-    > callback = nullptr)
+    std::function<void(
+      std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>)> callback =
+    nullptr)
```

Comment:
Then please update your working copy. E.g. the `uncrustify` repo is more then two months old.